models/bert/args.py

```diff
@@ -7,6 +7,7 @@ def get_args():
     parser = models.args.get_args()
 
     parser.add_argument('--model', default=None, type=str, required=True)
+    parser.add_argument('--hf-checkpoint', default=False, type=bool)
```

Comment:
Rather than adding a new flag, can't we just add the pre-trained model weights to https://git.uwaterloo.ca/jimmylin/hedwig-data? That way everyone would be able to get the models running out of the box

Reply:
Yes, that would definitely be better! What were the pre-trained models? Where they the [checkpoints](https://huggingface.co/transformers/pretrained_models.html) from hugging face or seperatly trained? I tried using `bert-base-uncased` and got results comparable to the ones in the paper.

Reply:
Yup, they are the checkpoints from hugging face. Do you have access to any of the DSG servers? We already have the exact pre-trained models we used for our experiments.

Reply:
No I don't have access unfortunately. Is there anyway I could download them into the notebook or maybe just wait until they are added into the `hedwig-data` repo so they are included when cloning.

Reply:
I've added the pre-trained weights to our data repo: https://git.uwaterloo.ca/jimmylin/hedwig-data